services/docker-compose-compass.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   compass:
     restart: always
-    image: iqtlabs/pibackbone-compass:latest
+    image: iqtlabs/pibackbone-compass:${SERVICES_IMAGE_TAG:-latest}
```

Comment:
@meadej this was a mistake, this should be `stable` like the others

Reply:
@cglewis - Understood. Changed and pushed 👍